### DISCUSSION TITLE
I deleted the NPC Lol (Fox)

### DISCUSSION THREAD

**Discussion by NX721** [Prodigy-Hacking/ProdigyMathGameHacking#864] [(C) Suggestions]
Ik what is missing, was just expeirementing and then it deleted the NPC
![image](https://user-images.githubusercontent.com/77288637/115255257-146fb500-a0fc-11eb-86ec-8e78f2d10bec.png)

**Comment by NX721** [comment]
Oh Shit 
![image](https://user-images.githubusercontent.com/77288637/115255562-631d4f00-a0fc-11eb-9d31-62fe19dacf03.png)

**Reply by ghost** [reply]
@Time-dragon I agree I hate it when people swear

**Reply by ghost** [reply]
!!!

**Comment by ghost** [comment]
I dont see anything bad happening

**Reply by ghost** [reply]
@BurnedWaffle HE HAS FOX ARMOUR AND HES INTERING THE BUNNY PLACE
